pkgs/by-name/ze/zed-editor/0001-generate-licenses.patch

```diff
@@ -14,4 +14,8 @@ index 43b2f5c458..c740a3afa2 100755
 -fi
```

Comment:
Is this line supposed to be removed?

Reply:
Yes, see the patch before this commit. I didn't change this line.

Reply:
(as to why this line is removed at all: it disables checking for a precise version of cargo-about so we can use the one provided in nixpkgs)